app/javascript/packages/document-capture/components/file-input.jsx

```diff
@@ -194,17 +195,29 @@ function FileInput(props, ref) {
   }
 
   /**
-   * @param {string} fileLabel String velue of the label for input to display
+   * @param {string} fileLabel String value of the label for input to display
    * @param {Blob|string|null|undefined} fileValue File or string for which to generate label.
+   * @return {{'aria-label': string} | {'aria-labelledby': string}}
```

Comment:
This'd be a good file to port to TypeScript, though having just opened the file myself, I see it's a big one, and probably not a task we'd want to undertake here 😅